src/CoreBundle/Contao/Hooks/FilterContentElementCallback.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * This file is part of MetaModels/core.
+ *
+ * (c) 2012-2025 The MetaModels team.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * This project is provided in good faith and hope to be usable by anyone.
+ *
+ * @package    MetaModels/core
+ * @author     Ingolf Steinhardt <info@e-spin.de>
+ * @copyright  2012-2025 The MetaModels team.
+ * @license    https://github.com/MetaModels/core/blob/master/LICENSE LGPL-3.0-or-later
+ * @filesource
+ */
+
+namespace MetaModels\CoreBundle\Contao\Hooks;
+
+use Contao\DC_Table;
+use MetaModels\CoreBundle\Contao\Hooks\AbstractContentElementAndModuleCallback;
```

Comment:
Why this line? We are in the same namespace and it seems superfluous.

Reply:
also superflous?
https://github.com/MetaModels/core/blob/76a55a20d55269cf29cc4be756970c9b8315491b/src/CoreBundle/Contao/Hooks/ContentElementCallback.php#L23